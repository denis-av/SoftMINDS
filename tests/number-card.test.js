import React from 'react';
import NumberCard from '../components/number-card';

import renderer from 'react-test-renderer';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

test('renders correctly the snapshot', () => {
  const tree = renderer.create(<NumberCard />).toJSON();
  expect(tree).toMatchSnapshot();
});