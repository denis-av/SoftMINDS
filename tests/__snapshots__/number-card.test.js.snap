// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the snapshot 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": undefined,
        "borderRadius": 0,
        "height": 60,
        "justifyContent": "center",
        "width": 58,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "bold-font",
          "fontSize": 35,
        }
      }
    />
  </View>
</View>
`;
